# Perspectivas: experimental y teórica

In [ ]:
# Configuración: el código auxiliar está en el paquete notebooks/fnyp/
try:
    import fnyp
except ModuleNotFoundError:      # en Google Colab hay que traerse antes el repositorio
    !git clone -q https://github.com/jahernando/USC-FNyP.git
    import sys; sys.path.append('USC-FNyP/notebooks')
from fnyp.inicio import *

 Last version  Sun Sep 20 10:08:18 2026


## Objetivos

En el tema anterior vimos *qué* se mide: la sección eficaz, la vida media, el invariante
$s$, la fracción de desintegración. Este tema mira el mismo campo desde los dos lados
desde los que se trabaja en él:

* **La perspectiva experimental**: cómo se convierte una partícula en cuadrimomentos, cuadrivectores.
  - Cómo se producen las partículas que se estudian: los aceleradores.
  - Cómo interaccionan con la materia, que es lo único que permite detectarlas.
  - Qué firma deja cada una y con qué detectores se recoge.
  - Qué le ocurre al dato desde el sensor hasta la figura de un artículo.

* **La perspectiva teórica**: cómo se calcula ese mismo número.
  - ¿Cómo se describen los fermiones libres? La Ecuación de Dirac.
  - El corazón de la teoría es el calculo del elemento de matriz.
  - La interacción de cuatro cuerpos: la teoría de Fermi.
  - Los diagramas de Feynman. 

Se trata de una aproximación, de intentar comprender los elementos fundamentales si entrar en el detalle de cálculo.

## Una perspectiva experimental

Un experimento de partículas es una cadena: una máquina produce las partículas, éstas
dejan energía en un material, unos sensores recogen esa energía y un análisis la convierte

A partir de esas deposiciones reconstruimos, dependiendo del caso, la trayectoria, el
momento, la identidad de la partícula, y luego a partir de ellos la sección eficaz o la vida media o la fracción de desintegración.

La cadena tiene cuatro eslabones, y los vamos a recorrer por su orden:
| | |
|:--|:--|
| el acelerador **produce** las partículas | *¿de dónde salen, y con qué energía?* |
| la partícula **interacciona** con la materia | *¿cuánta energía deja, y de qué forma?* |
| el detector **recoge** esa señal | *¿qué señal deja cada partícula?* |
| el análisis **convierte** sucesos en observables | *¿cómo se pasa de señales a estimar un valor con su error?* |

### Aceleradores

**¿De dónde salen las partículas que chocan?**

En un acelerador solo se pueden acelerar partículas **cargadas y estables**: $p$,
$\bar{p}$, $e$, $e^+$ e iones. Se agrupan en **paquetes** (*bunches*) de
$\mathcal{O}(10^{11})$ partículas que circulan por un tubo en ultra-alto vacío, y sobre
ellos actúan tres sistemas:

* **cavidades de radiofrecuencia**, que aceleran con gradientes de potencial sincronizados
  con el paso del paquete;
* **imanes dipolares**, que curvan la trayectoria para mantenerla en la órbita;
* **imanes cuadripolares**, que enfocan el haz, porque los paquetes tienden a dispersarse.

Y hay dos familias: **lineales**, que el paquete recorre una sola vez, y **circulares**,
donde da $\mathcal{O}(10^5)$ vueltas ganando energía en cada una.

#### Aceleradores lineales

| | |
| :--: | :--: |
| <img src="./imgs/Linear_accelerator_animation_16frames_1.6sec.gif" width = 400 align="center"> | <img src="./imgs/det_slac_aerial.png" width = 200 align="center">|
| principio de un acelerador lineal [WK]| el linac de SLAC, 3 km |

El linac de SLAC (California), de $3$ km, aceleró electrones hasta $50$ GeV; con él se
se midió l sec. eff. de Mott (Nuclar) y se descubrió la estructura de partones del protón. Los aceleradores lineales son además el primer eslabón de cualquier complejo de
aceleradores, y la técnica de todos los aceleradores médicos.

#### Aceleradores circulares

| | |
|:-- | :-- |
| <img src="./imgs/det_CERN_accelerators_complex.png" width = 300 align="center"> | <img src="./imgs/det_LHC_aerial.png" width = 320 align="center"> |
| el complejo de aceleradores del CERN [[>]](https://www.youtube.com/watch?v=pQhbhpU9Wrg)|  vista aérea del LHC [CERN] |

La energía que alcanza un anillo la fija un campo magnético perpendicular y el radio
que se puede excavar, conocemos la relación del electro-magnetismo:

$$
p \, [\mathrm{TeV}] = 0.3 \; B \, [\mathrm{T}] \; \rho \, [\mathrm{km}]
$$

Es la **misma** fórmula con la que mediremos el momento de una traza dentro de un detector: la física es idéntica, y solo cambia quién es el dato y quién la incógnita. 

En el Tevatron,
$\rho = 1$ km y $B = 1.5$ T daban $\simeq 1$ TeV. Para subir hay que ir a imanes
**superconductores**: los dipolos del LHC operan a $8.3$ T y a una temperatura de $1.9$ K, más fríos que el espacio exterior.

##### El límite: la radiación de sincrotrón

Una carga acelerada radia, y una partícula que gira está acelerada. La energía radiada por
vuelta es

$$
\Delta E \propto \frac{1}{\rho} \, \frac{E^4}{m^4}
$$

y ese $m^4$ en el denominador es el hecho central de la física de aceleradores: **un
electrón radia $(m_p/m_e)^4 \approx 10^{13}$ veces más que un protón de la misma energía**.

Por eso los récords de energía son siempre de máquinas de protones, y por eso LEP —$e^+e$
en el mismo túnel de $27$ km que hoy ocupa el LHC— se quedó en $104$ GeV por haz mientras
que el LHC llega a $7$ TeV. Por eso también, si se quiere un colisionador $e^+e$ de alta
energía, o se hace lineal (ILC, CLIC) o se hace un anillo mucho mayor (FCC-ee, $91$ km).

<div class="admonition tip">
<p class="admonition-title"><b>[>] Taller — LEP frente a LHC: el precio de girar</b></p>

Para electrones, la energía radiada por vuelta es, en la práctica,

$$
\Delta E \, [\mathrm{keV}] = 88.5 \; \frac{E^4 \, [\mathrm{GeV}]}{\rho \, [\mathrm{m}]}
$$

y para otra partícula hay que multiplicar por $(m_e/m)^4$.

1. ¿Cuánta energía pierde por vuelta un electrón de LEP-II ($E = 100$ GeV,
   $\rho = 3096$ m)? ¿Qué fracción de su energía es?
2. ¿Y un protón del LHC ($E = 7$ TeV, $\rho = 2804$ m), en el mismo túnel?
3. LEP daba $11\,245$ vueltas por segundo. ¿Qué potencia había que reponer con las
   cavidades para mantener los dos haces, si cada uno lleva $\sim 10^{12}$ electrones?
4. ¿Qué energía podría alcanzar LEP si se le permitiera radiar $10$ veces más por vuelta?

</div>

In [2]:
# Radiacion de sincrotron por vuelta: DE[keV] = 88.5 E^4[GeV] / rho[m] * (m_e/m)^4
m_e, m_p = 0.511, 938.3                      # MeV

def sincrotron(E_GeV, rho_m, m_MeV=m_e):
    return 88.5 * E_GeV**4 / rho_m * (m_e / m_MeV)**4   # keV por vuelta

dE_lep = sincrotron(100., 3096.)                        # LEP-II, electrones
dE_lhc = sincrotron(7000., 2804., m_p)                  # LHC, protones

print('LEP-II (e,   100 GeV) : {:8.2f} keV/vuelta = {:.2f} GeV'.format(dE_lep, dE_lep*1e-6))
print('LHC    (p,      7 TeV): {:8.2f} keV/vuelta'.format(dE_lhc))
print('razon                 : {:.1e}'.format(dE_lep/dE_lhc))

LEP-II (e,   100 GeV) : 2858527.13 keV/vuelta = 2.86 GeV
LHC    (p,      7 TeV):     6.67 keV/vuelta
razon                 : 4.3e+05


<details>
<summary><b>Solución</b></summary>

1. $\Delta E = 88.5 \times 10^8 / 3096 = 2.9 \cdot 10^6$ keV $\approx 2.9$ **GeV por
   vuelta**: casi el $3\,\%$ de su energía, perdida y repuesta $11\,245$ veces por segundo.
   LEP era, en la práctica, una gigantesca lámpara de rayos X.
2. $88.5 \times 7000^4 / 2804 \times (0.511/938.3)^4 \approx 6.7$ **keV por vuelta**: un
   factor $4 \cdot 10^5$ menos, pese a que el protón tiene $70$ veces más energía.
3. $P = \Delta E \times f \times N = 2.9\ \mathrm{GeV} \times 1.12 \cdot 10^4\
   \mathrm{s^{-1}} \times 10^{12} \approx 3.3 \cdot 10^{16}$ GeV/s $\approx 5$ MW por haz,
   $\sim 10$ MW en total. Ese, y no otro, fue el motivo de cerrar LEP: la potencia de
   radiofrecuencia necesaria crece como $E^4$ y se vuelve inasumible.
4. Como $\Delta E \propto E^4$, un factor $10$ en la pérdida son $10^{1/4} = 1.8$ en la
   energía: apenas $180$ GeV por haz. **La cuarta potencia es una pared**: no se sale de
   ella gastando más, solo cambiando de partícula o de radio.

</details>

##### Anillos de almacenamiento y colisionadores

Un **anillo de almacenamiento** mantiene circulando dos haces en sentidos opuestos durante
horas. En determinados **puntos de colisión**, unos imanes comprimen los paquetes y los
cruzan, y ahí se instalan los experimentos.

En el LHC los paquetes de $\sim 10^{11}$ protones miden unos $40$ cm de largo y $1$ mm de
sección, que en el punto de colisión se aprieta hasta $\sim 10\ \mu\mathrm{m}$. Se cruzan cada $25$ ns —$40$ MHz—, y en cada cruce ocurren
decenas de colisiones protón-protón simultáneas (*pile-up*), de las que solo una suele
interesar.

Los parámetros que definen un colisionador son los dos que ya conocemos del tema anterior:
la **energía** $\sqrt{s}$, que fija qué se puede crear, y la **luminosidad**
$\mathcal{L}$, que fija cuánto se crea, la relación entre ambos es

$$
R = \sigma \mathcal{L}
$$,

donde $R$ es la razón de producción de interacciones en colisiones. $\mathcal{L}$ es la luminosidad, lo que nos indica la capacidad de la máquina para generar interacciones (el equivalente al flujo $\phi_a$ y número de blancos $N_b$ en los experimentos de blanco fijo). 

La luminosidad se mide en $1/b$. 

| colisionador | laboratorio | tipo | periodo | $\sqrt{s}$ | $\mathcal{L}$ (cm$^{-2}$s$^{-1}$) | descubrimientos |
| :-- | :-- | :-- | :-- | :-- | :-- | :-- |
| SPS | CERN | $p + \bar{p}$ | 1981-1984 | 315 GeV | $10^{30}$ | $W^\pm$, $Z$ |
| SLC | SLAC | $e^+ + e$ | 1988-1998 | 91 GeV | $10^{30}$ | (lineal) |
| LEP | CERN | $e^+ + e$ | 1989-2000 | 91-209 GeV | $10^{32}$ | $N_\nu = 3$, precisión del SM |
| HERA | DESY | $e + p$ | 1992-2007 | 320 GeV | $10^{31}$ | estructura del nucleón |
| PEP-II | SLAC | $e^+ + e$ | 1999-2008 | 10.5 GeV | $10^{34}$ | violación de CP en $B$ |
| Tevatron | Fermilab | $p + \bar{p}$ | 1987-2012 | 1.96 TeV | $4 \cdot 10^{32}$ | quark $t$ |
| LHC | CERN | $p + p$ | 2009- | 7-13.6 TeV | $2 \cdot 10^{34}$ | Higgs |
| SuperKEKB | KEK | $e^+ + e$ | 2018- | 10.6 GeV | $5 \cdot 10^{34}$ | (en marcha) |

**¿Qué ves?** Las máquinas de hadrones ganan en energía —son **máquinas de
descubrimiento**— y las de leptones, en limpieza: la colisión es entre partículas
elementales, con $\sqrt{s}$ conocido, y son **máquinas de precisión**. Y la luminosidad ha
subido cuatro órdenes de magnitud en treinta años.

#### Haces secundarios

No todo lo que se quiere lanzar contra algo es estable. Los haces de $\pi$, $K$, $\mu$ y
$\nu$ se fabrican en dos pasos: se golpea con un haz primario de protones un blanco fijo, y
de los restos se selecciona lo que interesa.

| |
|:--: |
| <img src="./imgs/det_nubeam_schematic.jpg" width = 480 align="center"> |
| esquema de un haz de neutrinos [Fermilab] |
| <img src="./imgs/det_nubeam_DUNE.jpg" width = 480 align="center"> |
| el haz de $\nu$ de Fermilab hacia DUNE (en construcción) [Fermilab]|

La selección la hacen imanes ajustados para dejar pasar solo una carga y un momento
determinados. Para un **haz de neutrinos** se deja además que los piones se desintegren en
vuelo, $\pi^+ \to \mu^+ + \nu_\mu$, en un túnel de desintegración, y después se absorben en
roca los muones y todo lo demás: lo único que atraviesa cientos de metros de tierra es el
neutrino. Es la técnica de los experimentos de oscilaciones.

### ¿Qué llega al detector?

**¿Qué particulas se detectan pocas partículas?**

De las partículas que aparecen en una interacción, solo un puñado vive lo suficiente para recorrerlo y dejar
una firma directa, entre ellas las estables:

* Las **estables**: $p$, $e$, $\nu$.

* Las de **vida media larga**, que recorren $\mathcal{O}$(m) antes de desintegrarse:
  $\mu$, $\gamma$, $K^\pm$, $K^0_L$, $\pi^\pm$, $n$.

Estas se detectan como **trayectorias** o como **depósitos de energía**. Su identidad se
deduce de cómo penetran, cuánto ionizan, cuánta luz producen y en qué se desintegran.


Las demás —la inmensa mayoría— no llegan nunca a un sensor. Se detectan de forma
indirecta, y de dos maneras según lo que vivan:

* Si la partícula **vuela** $\mathcal{O}$(mm–cm) antes de desintegrarse ($\tau$, hadrones
  con $b$ o $c$, como el $B^0_s$), los detectores de trazas (o vértices) precisos reconstruyen su **vértice de
  desintegración**, separado del punto de la colisión.

* Si su vida es tan corta que se desintegra donde se produjo ($Z$, $W$, $\rho$, $\Delta$),
  solo queda el rastro de sus productos: aparece como una **resonancia** en la sección
  eficaz frente a la energía, o como un pico en la **masa invariante** de lo que sale.

Las dos cosas —el pico del $Z$ en OPAL, el pico del $B^0_s$— ya las vimos en el tema
anterior. Ahora vamos a ver de dónde salen los cuadrimomentos con los que se construyen
esos picos.

### Interacción con la materia

**¿Cómo se ve algo que no podemos alcanzar?**

Una partícula solo es visible si deja parte de su energía en el material que atraviesa. Y
la deja de tres maneras distintas, según lo que sea:

| partículas | mecanismo dominante | escala característica |
|:--|:--|:--|
| cargadas ($e$, $\mu$, $p$, $\pi^\pm$, $K^\pm$) | **ionización** y excitación del medio | $\mathrm{d}E/\mathrm{d}x$ |
| $e$ y $\gamma$ de alta energía | **bremsstrahlung** y **producción de pares** | longitud de radiación, $X_0$ |
| hadrones | **interacción fuerte** con los núcleos | longitud de interacción, $\lambda_I$ |

Los tres mecanismos coexisten: un electrón ioniza *y* radia, un pión ioniza *y* choca con
un núcleo. Lo que cambia es cuál domina, y eso depende de la partícula, de su energía y
del material.

#### Las partículas cargadas: ionización

Una partícula cargada que atraviesa un medio interacciona electromagnéticamente con los
electrones de los átomos: los excita o los arranca (**ioniza**). En cada choque pierde muy
poca energía —arrancar un electrón cuesta $\mathcal{O}(10)$ eV— pero hay muchísimos
choques, y el efecto acumulado es una pérdida continua y predecible.

La mayor parte de los detectores de trayectorias se basan en recoger esos electrones de
ionización.

La pérdida media de energía por unidad de camino la da la fórmula de **Bethe-Bloch**
(años 30):

$$
\frac{1}{\rho}\frac{\mathrm{d}E}{\mathrm{d}x} \simeq - K \frac{Z}{A \, \beta^2}
\left[\ln \left( \frac{2 m_e c^2 \beta^2 \gamma^2}{I_e}\right) - \beta^2\right]
$$

donde $K = 4\pi \hbar^2 \alpha^2 N_A / m_e = 0.307$ MeV cm$^2$/mol, $(A, Z)$ caracterizan
el material, $\rho$ es su densidad, $I_e$ su potencial medio de ionización, y $\beta,
\gamma$ son los de la partícula incidente.

Conviene leerla despacio, porque dice tres cosas nada obvias:

* **no depende de la masa** de la partícula incidente, solo de su velocidad $\beta$ y de su
  carga. Un $\pi^\pm$ y un $p$ con la misma $\beta$ pierden lo mismo;

* va como $1/\beta^2$: cuanto **más despacio**, más ioniza;

* dividida por la densidad, $\frac{1}{\rho}\frac{\mathrm{d}E}{\mathrm{d}x}$, es casi la
  misma para todos los materiales ($Z/A \simeq 0.5$ salvo en el hidrógeno).

Esto hace que la curva sea prácticamente universal en función de $\beta\gamma$:

| |
|:--|
|<img src="./imgs/det_eneloss_mip.jpg" width = 400 align = "center">|
|pérdida de energía por ionización [PDG]|

**¿Qué ves?** La curva tiene tres regiones, y cada una tiene una consecuencia práctica:

* **$\beta\gamma \lesssim 1$**: la pérdida crece como $1/\beta^2$. Como la partícula se va
  frenando, ioniza cada vez más, y el depósito es máximo **justo antes de pararse**: es el
  **pico de Bragg**, en el que se basa la hadronterapia.

* **$\beta\gamma \sim 3$–$4$**: el mínimo. Una partícula ahí se llama **mip** (*minimum
  ionizing particle*) y pierde $\simeq 2$ MeV cm$^2$/g, casi independientemente del
  material. En hierro, $\simeq 11$ MeV/cm. Casi todas las partículas cargadas de un
  experimento de altas energías están en esta región o por encima.

* **$\beta\gamma \gtrsim 100$**: la pérdida vuelve a subir, ahora logarítmicamente. Es la
  *subida relativista*, y es lo que permite identificar partículas midiendo
  $\mathrm{d}E/\mathrm{d}x$.

La distancia que recorre una partícula hasta detenerse es su **rango**. Se obtiene
integrando $\mathrm{d}E/\mathrm{d}x$ desde su energía inicial hasta cero.

#### Electrones y fotones

Un electrón, además de ionizar, **radia** fotones al ser desviado por el campo del núcleo.
Es la radiación de frenado o **bremsstrahlung**:

$$
e + (A, Z) \to e + \gamma + (A, Z)
$$

La sección eficaz se calcula en QED y va como $\sigma_b \propto E/m^2$: la **masa al
cuadrado en el denominador** es lo decisivo. Un muón radia $(m_e/m_\mu)^2 \approx 1/43000$
veces menos que un electrón, y por eso por debajo de $\mathcal{O}(100)$ GeV los muones solo
ionizan —y por eso atraviesan el detector entero—.

Por encima de una **energía crítica** $E_c$ la radiación supera a la ionización:

$$
E_c \simeq \frac{610 \; \mathrm{MeV}}{Z + 1.24} \quad \text{(sólidos)}
$$

que vale $\simeq 22$ MeV en hierro y $\simeq 7$ MeV en plomo. Por encima de unas pocas
decenas de MeV, un electrón en un material denso esencialmente **solo radia**.

Por encima de $E_c$ la pérdida es **proporcional a la energía**, y eso hace que la caída
sea exponencial en el camino:

$$
\frac{\mathrm{d}E}{\mathrm{d}x} = - \frac{E}{X_0}
\;\; \to \;\;
E(x) = E_0 \, e^{-x/X_0}
$$

La constante $X_0$ es la **longitud de radiación**: la distancia en la que un electrón
pierde, en promedio, una fracción $1 - 1/e$ de su energía. Depende solo del material
($X_0 = 1/(n\sigma_b)$, con $n$ la densidad de núcleos) y es *la* escala con la que se
diseñan los calorímetros:

| H$_2$ (gas) | H$_2$O | C (grafito) | Xe (líq.) | Fe | PbWO$_4$ | Pb |
| :-- | :-- | :-- | :-- | :-- | :-- | :-- |
| 7 km | 36.1 cm | 18.8 cm | 2.87 cm | 1.76 cm | 0.89 cm | 0.56 cm |

| |
| :-- |
|<img src="./imgs/det_AB1.1_electron_xsec_Pb.png" width = 400 align = "center">|
|pérdida de energía de los electrones en plomo [AB1.1]|

Los **fotones** no ionizan: desaparecen o se dispersan en un único suceso. Cuál domina
depende de la energía:

| energía | proceso |
|:--|:--|
| $\lesssim 100$ keV | efecto **fotoeléctrico**, $\gamma + \mathrm{átomo} \to e + \mathrm{ion}$ |
| $\sim$ MeV | dispersión **Compton**, $\gamma + e \to \gamma + e$ |
| $\gtrsim 10$ MeV | **producción de pares**, $\gamma + (A,Z) \to e^+ + e + (A,Z)$ |

| |
| -- |
| <img src="./imgs/det_AB1.1_gamma_xsec_Pb.png" width = 340 align = "center">|
| sección eficaz de fotones en plomo [AB1.1]|

A alta energía, donde domina la producción de pares, la sección eficaz se puede escribir en
términos de la **misma** $X_0$ que la de los electrones:

$$
\sigma_\gamma \simeq \frac{7}{9} \frac{1}{n X_0}
\quad \to \quad
\lambda_\gamma = \frac{1}{n \sigma_\gamma} = \frac{9}{7} X_0
$$

donde $\lambda_\gamma$ es el **camino libre medio** del fotón, y la intensidad de un haz de
fotones cae como $I(x) = I_0 \, e^{-x/\lambda_\gamma}$.

Que una sola constante, $X_0$, gobierne a la vez la radiación de los electrones y la
conversión de los fotones no es casualidad: los dos procesos son el mismo diagrama de QED
con las patas cambiadas de sitio. Y es lo que hace posible el calorímetro.

**La cascada electromagnética.** Un electrón de alta energía radia un fotón; el fotón se
convierte en un par $e^+ e$; cada uno de ellos vuelve a radiar. Cada $X_0$, el número de
partículas se duplica y la energía por partícula se divide por dos. El proceso se detiene
cuando la energía cae por debajo de $E_c$ y la ionización toma el relevo.

| | |
| :--: | :--: |
| <img src="./imgs/det_gamma_cascade.png" width = 300 align = "center">| <img src="./imgs/det_gamma_cascade_real.png" width = 300 align = "center"> |
| esquema de una cascada [AB]| cascada real [AB]|

Dos consecuencias, las dos importantes:

* el máximo de la cascada está a $t_{max} \simeq \ln(E_0/E_c)$ longitudes de radiación: la
  profundidad necesaria crece **logarítmicamente** con la energía;

* el número total de partículas producidas es proporcional a $E_0$, y por tanto la señal
  recogida también. Eso es un **calorímetro**.

#### Los hadrones

Un hadrón cargado ioniza como cualquier partícula cargada, pero además choca con los
núcleos del material por **interacción fuerte**, y en ese choque se rompe y produce más
hadrones. La escala aquí no es $X_0$ —que es electromagnética— sino la **longitud de
interacción nuclear** $\lambda_I$, que es la distancia media entre choques fuertes:

| | Fe |
|:-- | :-- |
| $\lambda_I$ | 16.8 cm |
| $X_0$ | 1.76 cm |

En hierro $\lambda_I / X_0 \approx 10$: **un hadrón penetra mucho más que un electrón de la
misma energía**. De ahí el orden de las capas de un experimento.

La **cascada hadrónica** es mucho más irregular que la electromagnética:

* en ella se producen $\pi^0$, que se desintegran $\pi^0 \to \gamma + \gamma$ y arrancan una
  cascada *electromagnética* dentro de la hadrónica. La fracción de energía que se va por
  esa vía fluctúa suceso a suceso;

* parte de la energía se pierde en romper y excitar núcleos, y no produce señal: es la
  energía **invisible**.

Las dos cosas empeoran la resolución de los calorímetros hadrónicos en un factor $\sim 5$
frente a los electromagnéticos, como veremos.

<div class="admonition tip">
<p class="admonition-title"><b>[>] Taller — ¿Cuánto hierro para parar cada partícula?</b></p>

Tienes cuatro partículas de $10$ GeV y un bloque de hierro ($X_0 = 1.76$ cm,
$\lambda_I = 16.8$ cm, $E_c = 22$ MeV, $\mathrm{d}E/\mathrm{d}x|_{mip} \simeq 11$ MeV/cm).
Estima el espesor que necesitas para absorber cada una:

1. un **electrón**. Una cascada electromagnética está contenida al 95 % en unas
   $t_{max} + 10$ longitudes de radiación.
2. un **pión** cargado. Una cascada hadrónica necesita unas $6\,\lambda_I$.
3. un **muón**, que no hace cascada: solo ioniza.
4. un **neutrino** (lo calculaste en el tema anterior).

¿Qué te dice el resultado sobre el orden en que hay que poner los detectores?

</div>

<details>
<summary><b>Solución</b></summary>

1. $t_{max} = \ln(10^4/22) \approx 6.1$, luego $\approx 16 \, X_0 \approx 28$ cm. **Unos
   30 cm.**
2. $6 \times 16.8 \approx 100$ cm. **Un metro**, tres veces más que el electrón.
3. $10^4\ \mathrm{MeV} \, / \, 11\ \mathrm{MeV/cm} \approx 900$ cm. **Unos 9 metros** —y es
   una sobreestimación, porque solo está en el mínimo durante parte del recorrido; el
   rango real es algo menor. Aun así, un orden de magnitud más que el pión.
4. $\lambda \sim$ años luz de material: no se para. Se detecta, con suerte, la
   interacción de uno entre $10^{19}$.

**El orden de las capas es exactamente el orden de esta lista**: primero las trazas (que
no deben estropear nada), luego el calorímetro electromagnético, que para $e$ y $\gamma$
en $\sim 30$ cm; detrás el hadrónico, que necesita un metro; y al final, fuera de todo, las
cámaras de muones, porque lo único que puede llegar hasta ahí es un muón. El neutrino se
va, y su rastro es la energía que falta.

</details>

### Detección e identificación de partículas

**¿Cómo se sabe que aquello era un electrón y no un pión?**

No hay ningún detector que diga "electrón". Lo que hay es una combinación de firmas: si
dejó traza o no, dónde depositó su energía y hasta dónde llegó. Con tres capas basta para
separar todo lo que sale de una colisión:

| partícula | traza (campo $B$) | calorímetro EM | calorímetro hadrónico | cámaras de $\mu$ |
|:--|:--:|:--:|:--:|:--:|
| $e$, $e^+$ | **sí** | **para** | — | — |
| $\gamma$ | no | **para** | — | — |
| $\mu$ | **sí** | deja poco | deja poco | **sí** |
| $\pi^\pm$, $K^\pm$, $p$ | **sí** | empieza | **para** | — |
| $n$, $K^0_L$ | no | empieza | **para** | — |
| $\nu$ | no | — | — | — |

Las dos primeras filas se separan por la traza; la tercera y la quinta, por dónde acaba la
partícula. El neutrino se identifica *por lo que falta*: el desbalance de momento
transverso, $p_T^{miss}$.

Este es el corte de CMS, y se lee exactamente como la tabla anterior:

|  |
|:-- |
| <img src="./imgs/det_CMS_subdetectors.png" width = 500 align = "center"> |
| sección transversal del *Compact Muon Solenoid* ([CMS](https://cms.cern/detector)) del LHC|

Además de esta separación gruesa, hay medidas que afinan la identificación:

* la razón $E/p$ entre la energía del calorímetro y el momento de la traza vale $\simeq 1$
  para un electrón y $< 1$ para un hadrón;
* el $\mathrm{d}E/\mathrm{d}x$ medido en el detector de trazas separa partículas de la
  misma carga y distinto momento (lo veremos en la TPC de ALICE);
* la luz **Cherenkov** da $\beta$, y con el momento de la traza, la masa;
* los **vértices desplazados** identifican lo que vivió lo justo para volar unos mm.

#### El momento transverso perdido

**¿Cómo se mide lo que no deja ninguna señal?**

En un colisionador de hadrones lo que choca no son los protones, sino sus constituyentes,
cada uno con una fracción desconocida del momento del protón. El momento inicial **a lo
largo del haz no se conoce**; el **transverso** al haz, sí: es cero. Y si lo era al
principio, lo es al final. Por eso el plano transverso es el plano natural del análisis: es
el único en el que se puede imponer conservación del momento.

Si al sumar el momento transverso de todo lo reconstruido el resultado **no** es cero, algo
se ha ido sin dejar señal, y lo que falta es justo su momento transverso:

$$
{\bf p}^{miss}_T \equiv - \sum_{\mathrm{visible}} {\bf p}_{T, i},
\qquad E^{miss}_T = |{\bf p}^{miss}_T|
$$

| |
|:--:|
| <img src="./imgs/det_ptmiss_scheme.png" width = 300 align = "center"> |
| el momento que falta es el del neutrino, en $W \to e + \bar{\nu}$ |

El $E^{miss}_T$ es la firma de las partículas **"fantasma"**: las que atraviesan el
detector entero sin interaccionar. En el Modelo Estándar solo hay una, el **neutrino**;
cualquier otra sería nueva física —materia oscura, partículas supersimétricas—, y daría
exactamente la misma señal. Buscar "algo que se escapa" es, literalmente, buscar
$E^{miss}_T$.

No es un subdetector más: es una resta sobre **todo** el suceso. Por eso los experimentos
se construyen **herméticos** (de ahí el $4\pi$): un agujero, una grieta entre calorímetros
o un jet mal medido producen un $E^{miss}_T$ falso.

De la partícula fugitiva se obtiene **solo su $p_T$**, no su componente longitudinal: no
hay cuadrimomento, y por tanto tampoco masa invariante. En su lugar se usa la **masa
transversa**; por ejemplo, en $W \to e + \bar{\nu}$:

$$
m^2_T = 2 \, p^e_T \, p^{miss}_T \, (1 - \cos \Delta \phi)
$$

con $\Delta\phi$ el ángulo entre ambos en el plano transverso. Se cumple $m_T \leq m_W$, y
la distribución tiene un **borde** (pico jacobiano) en $m_W$: así se descubrió el $W$ en el
SPS (1983) y así se sigue midiendo su masa en el LHC.

**Un caso distinto: $e^+ + e$.** Allí se conoce el estado inicial entero, así que se puede
usar la **energía perdida**, no solo la transversa. El ejemplo clásico es la anchura
invisible del $Z$ en LEP: lo que no se ve en ninguna parte mide el número de familias de
neutrinos, $N_\nu = 3$.

### Los detectores

#### Detectores de trazas: gaseosos

Recogen los electrones de ionización que la partícula deja a su paso en un gas noble (Ar,
Xe), a razón de un par ion-electrón cada $\sim 30$ eV depositados.

El principio es siempre el mismo: un campo eléctrico ${\bf E}$ hace **derivar** esos
electrones hasta un ánodo, donde su carga se amplifica y se lee. Se trabaja en régimen
proporcional —la señal es proporcional a la carga original, sin avalancha descontrolada—,
lo que permite medir también cuánto ionizó la partícula.

Hay muchas variantes: cámaras de hilos, cámaras proporcionales multihilo (MWPC, Charpak,
Nobel 1992), cámaras de deriva y cámaras de proyección temporal (TPC).

##### Cámaras de proyección temporal (TPC)

Una TPC es un gran volumen de gas, normalmente un barril, con un cátodo de alto voltaje en
el centro y dos tapas como ánodos.

La idea es elegante: las dos coordenadas transversas, $(r, \phi)$ o $(x, y)$, las da el
**punto** del ánodo segmentado donde llega la carga; la tercera, $z$, la da el **tiempo**
de llegada, porque los electrones derivan con velocidad conocida. De ahí el nombre. Con un
solo detector se reconstruye la trayectoria en tres dimensiones.

| | |
|-- | -- |
| <img src="./imgs/det_ALICE_TPC_scheme.png" width = 400 align = "center">| <img src="./imgs/det_ALICE_ppevent.jpeg" width = 400 align = "center"> |
| [TPC de ALICE](https://alice-collaboration.web.cern.ch/menu_proj_items/tpc) | suceso reconstruido en ALICE [ALICE]|

Las TPC se usan también, con gas o líquido noble, en los experimentos de materia oscura
(XENON, LZ) y de neutrinos (EXO, NEXT, y las TPC de argón líquido de DUNE).

Además de la trayectoria, una TPC mide la **ionización** de cada traza, y con ella
identifica la partícula: para un mismo momento, cada masa da un $\mathrm{d}E/\mathrm{d}x$
distinto, porque lo que entra en Bethe-Bloch es $\beta$.

| |
|:--|
|<img src="./imgs/det_eloss_alice_tpc.png" width = 400 align = "center">|
| pérdida de energía medida en la TPC de ALICE [ALICE]|

**¿Qué ves?** Cada banda es una especie de partícula ($e$, $\pi$, $K$, $p$, $d$). La
fórmula de Bethe-Bloch da el **valor medio**; la anchura de cada banda es la fluctuación
en el número de colisiones con los electrones del medio (distribución de Landau, con una
cola larga hacia arriba). Donde las bandas se cruzan, la identificación falla.

##### Detectores de silicio

Son el mismo principio, en estado sólido. Se parte de una oblea de silicio de $\sim 300\,
\mu\mathrm{m}$ de espesor en la que se dopan tiras (*strips*) tipo $p$ separadas $\sim 50\,
\mu\mathrm{m}$, formando uniones $pn$ que se polarizan en inversa para vaciarlas de
portadores libres.

| |
| :-- |
<img src="./imgs/det_silicon_detector.png" width = 400 align = "center">
|esquema de un sensor de micro-tiras de silicio|

Al pasar una partícula cargada crea pares electrón-hueco, y aquí está la ventaja: en
silicio cuesta $3.6$ eV crear un par, casi diez veces menos que ionizar un gas. Mucha más
señal, en mucho menos espesor.

El precio es el coste y la cantidad de canales de electrónica; la ganancia, una precisión
de $\sim 10\, \mu\mathrm{m}$ por punto, suficiente para separar el vértice de la colisión
del vértice de desintegración de una partícula que ha volado $\sim 1$ mm.

| | |
|-- | -- |
| <img src="./imgs/det_DELPHI_vertex.jpeg" width = 350 align = "center">| <img src="./imgs/det_DELPHI_bdecay.gif" width = 350 align = "center"> |
|detector de vértices de DELPHI | vista transversa de un suceso [DELPHI]|

Eso es lo que permite **etiquetar** hadrones con $b$ o $c$, y con ello casi toda la física
del quark top, del Higgs a $b \bar b$ y de la violación de CP. ATLAS, CMS y LHCb los
usan hoy; se desarrollaron en los años 80 y fueron decisivos en LEP, BaBar y Belle.

##### Medir el momento: el campo magnético

Un detector de trazas da la trayectoria, no el momento. El momento aparece cuando se
sumerge todo el detector en un **campo magnético**: la partícula cargada describe una
hélice, y su curvatura en el plano perpendicular a ${\bf B}$ mide su momento:

$$
p_T \equiv p \, \sin \theta = 0.3 \, B \, \rho
$$

con $p_T$ en GeV, $B$ en T y el radio de curvatura $\rho$ en m. A $p_T$ se le llama
**momento transverso**, y es la cantidad que se usa en un colisionador porque, a diferencia
del momento longitudinal, se conserva de forma útil: los restos del protón se van por el
tubo del haz y no se ven.

Es la fórmula con la que se dimensiona un anillo de aceleración, leída al revés: allí se
fijaba la energía y se despejaba $B\rho$; aquí se miden $B$ y $\rho$ y se despeja el
momento. El signo de la curvatura da además la **carga** de la partícula.

<div class="admonition tip">
<p class="admonition-title"><b>[>] Taller — ¿Hasta qué momento se puede medir una traza?</b></p>

El detector de trazas de CMS tiene un radio de $R = 1.1$ m y está dentro de un solenoide
de $B = 3.8$ T. Cada punto de la traza se mide con una precisión de $\sim 10\,
\mu\mathrm{m}$.

Lo que se mide de verdad no es el radio, sino la **sagita**: la separación entre la
trayectoria curva y la cuerda que une sus extremos, $s \simeq L^2/(8\rho)$, con $L$ la
longitud de la traza.

1. ¿Cuál es el radio de curvatura de una partícula de $p_T = 1$ GeV? ¿Y de $100$ GeV?
2. Calcula la sagita de la de $100$ GeV a lo largo del detector entero.
3. Repite para $p_T = 1$ TeV. ¿Qué le pasa a la resolución $\sigma_{p_T}/p_T$ cuando sube
   el momento? ¿Por qué se dice que un tracker mide bien las partículas *lentas*?

</div>

<details>
<summary><b>Solución</b></summary>

1. $\rho = p_T / (0.3 B) = 1/(0.3 \times 3.8) = 0.88$ m para $1$ GeV —se curva dentro del
   propio detector, incluso puede no llegar al calorímetro— y $\rho = 88$ m para $100$ GeV:
   ochenta veces el tamaño del detector, casi una recta.
2. $s = L^2/(8\rho) = 1.1^2/(8 \times 88) = 1.7 \cdot 10^{-3}$ m $= 1.7$ mm. Frente a los
   $10\ \mu\mathrm{m}$ de cada punto, se mide con holgura.
3. A $1$ TeV, $\rho = 877$ m y $s = 172\ \mu\mathrm{m}$: solo $17$ veces la precisión de un
   punto. Como $s \propto 1/p_T$ y el error en la sagita es fijo,

   $$\frac{\sigma_{p_T}}{p_T} = \frac{\sigma_s}{s} \propto p_T$$

   la resolución **empeora linealmente con el momento**. Es lo contrario de un
   calorímetro, donde $\sigma_E/E \propto 1/\sqrt{E}$ mejora al subir la energía. Por eso
   en un experimento se mide el momento con la traza a baja energía y la energía con el
   calorímetro a alta energía, y por eso se pelea por campos magnéticos grandes y
   detectores grandes: $s \propto B L^2$.

</details>

#### Calorímetros

Un calorímetro **destruye** la partícula para medirla: provoca una cascada de partículas hijas y recoge una
señal proporcional al número de partículas producidas, que es proporcional a la energía
incidente. A diferencia de las trazas, funciona también con partículas **neutras**.

Un **calorímetro electromagnético** se construye con material de $X_0$ pequeño para que la
cascada quepa. Hay dos diseños:

* **de muestreo**: capas alternas de material pasivo denso (Pb) donde se desarrolla la
  cascada, y material activo donde se lee la señal (centelleador, gas, Ar líquido). Es el
  caso de ATLAS (Pb + Ar líquido);

* **homogéneo**: un solo material que es a la vez absorbente y activo. Es el caso de CMS,
  con cristales de PbWO$_4$ ($X_0 = 0.89$ cm).

La resolución está limitada por las **fluctuaciones estadísticas** del número de partículas
de la cascada, $N \propto E$, de donde $\sigma_E / E \propto \sqrt{N}/N = 1/\sqrt{E}$:

$$
\frac{\sigma_E}{E} \sim \frac{3 - 10 \, \%}{\sqrt{E \; \mathrm{(GeV)}}}
$$

Los **centelleadores** son el material activo más común: contienen moléculas que se
excitan al paso de la partícula cargada y al desexcitarse emiten luz visible, un fotón por
cada $\sim 100$ eV depositados. Esa luz se lleva a un fotosensor. Los hay orgánicos
(plásticos, rápidos y baratos) e inorgánicos (cristales, densos y con mucha luz).

<div class="admonition tip">
<p class="admonition-title"><b>[>] Taller — ¿Cómo de gordo tiene que ser un calorímetro?</b></p>

Los cristales de PbWO$_4$ del calorímetro de CMS miden $23$ cm de largo, con
$X_0 = 0.89$ cm y $E_c \simeq 10$ MeV.

La cascada de un electrón alcanza su máximo a $t_{max} \simeq \ln(E_0/E_c)$ longitudes de
radiación, y hacen falta unas $10$–$15$ $X_0$ más para recoger la cola.

1. ¿A qué profundidad está el máximo de la cascada de un electrón de $100$ GeV? ¿Y cuánto
   cristal hace falta para contenerla? Compara con los $23$ cm reales.
2. ¿Y para un electrón de $1$ TeV? ¿Cuánto habría que alargar el cristal?
3. Un hadrón de $100$ GeV necesita $\sim 6 \lambda_I$, y en hierro $\lambda_I = 16.8$ cm.
   ¿Por qué el calorímetro hadrónico de un experimento es siempre mucho más voluminoso —y
   más caro— que el electromagnético?

</div>

<details>
<summary><b>Solución</b></summary>

1. $t_{max} = \ln(10^5/10) = \ln 10^4 \approx 9.2$, es decir $\approx 8$ cm de cristal.
   Sumando la cola, $\approx 22$–$25\,X_0 \approx 20$–$22$ cm: justo los $23$ cm de CMS.
   El detector está diseñado exactamente en este límite.
2. $t_{max} = \ln(10^6/10) = 11.5\,X_0$: **solo 2.3 $X_0$ más**, unos $2$ cm. La
   profundidad crece como $\ln E$, así que un calorímetro que vale para $100$ GeV vale casi
   igual para $1$ TeV. Es la gran virtud de la calorimetría, y la razón de que sea la
   técnica de alta energía por excelencia.
3. Porque la escala es otra: $6 \times 16.8 \approx 1$ m de hierro frente a $20$ cm de
   cristal, con un volumen que crece con el cubo del radio. Y además, con peor resolución
   (ver abajo). Es el subdetector más masivo y caro de un experimento.

</details>

##### Calorímetros hadrónicos

Miden la energía de los hadrones —cargados y neutros— mediante la cascada hadrónica. Se
construyen casi siempre por muestreo, con capas de material denso y material activo: el de
ATLAS alterna hierro y plástico centelleador.

La resolución es mucho peor que la electromagnética, por lo que ya vimos: la fracción de
energía que se va a la componente electromagnética (vía $\pi^0$) fluctúa, y hay energía
invisible en la rotura de los núcleos:

$$
\frac{\sigma_E}{E} \sim \frac{50 \, \%}{\sqrt{E \; \mathrm{(GeV)}}}
$$

Un hadrón de $100$ GeV genera una cascada de unos $2$ m de profundidad y $50$ cm de anchura:
de ahí el tamaño del subdetector.

#### Detectores Cherenkov

Cuando una partícula cargada atraviesa un medio **más deprisa que la luz en ese medio**,
$v > c/n$, la polarización del medio no se recompone a tiempo y emite un frente de onda
coherente: la **luz Cherenkov**, en un cono de semiángulo

$$
\cos \theta = \frac{1}{\beta \, n}
$$

| |
| :-- |
<img src="./imgs/det_cherenkov_schematic.png" width = 300 align = "center">
|esquema de la emisión de luz Cherenkov|

Detectar el cono da dos cosas a la vez: la **dirección** de la partícula, $\hat{\bf v}$, y
su **velocidad** $\beta$. Combinada con el momento de la traza, da la **masa**: es la
técnica de identificación de los RICH de LHCb.

Y hay un uso más, completamente distinto: los **detectores de neutrinos**. Con
$\sigma \sim 10^{-38}$ cm$^2$, para ver neutrinos hace falta una masa enorme, y el agua es
lo único que se puede permitir en esas cantidades.

| | |
| :-- | :-- |
| <img src="./imgs/det_SK_image.jpeg" width = 350 align = "center"> | <img src="./imgs/det_SK_ne_ring.jpeg" width = 300 align = "center"> |
|interior de Super-Kamiokande | anillo Cherenkov de una interacción de $\nu_e$ [SK]|

Super-Kamiokande es un cilindro de $50$ kilotoneladas de agua ($n = 1.33$), de $41$ m de
diámetro y $40$ m de altura, con $13\,000$ PMT en las paredes. Detecta el $e$ o el $\mu$
que produce el neutrino al interaccionar, $\nu_e (\nu_\mu) + N \to e (\mu) + X$, por el
anillo de luz Cherenkov que proyectan. Un anillo nítido es un $\mu$; uno difuso, un $e$,
porque el electrón hace cascada. Es el mismo detector con el que se buscó la
desintegración del protón.

<div class="admonition tip">
<p class="admonition-title"><b>[>] Taller — El umbral Cherenkov en agua</b></p>

En agua, $n = 1.33$.

1. ¿Cuál es la velocidad mínima $\beta$ para emitir luz Cherenkov? ¿Y el $\gamma$
   correspondiente?
2. ¿Qué energía cinética mínima necesita un electrón ($m = 0.511$ MeV)? ¿Y un muón
   ($m = 105.7$ MeV)?
3. ¿Cuál es la apertura máxima del cono?
4. Super-Kamiokande detecta neutrinos solares de $\sim 7$ MeV por el electrón que
   dispersan. ¿Podría detectar del mismo modo neutrinos de esa energía que produjeran un
   muón?

</div>

<details>
<summary><b>Solución</b></summary>

1. $\beta_{min} = 1/n = 0.752$, y $\gamma_{min} = 1/\sqrt{1 - \beta^2} = 1.52$.
2. $E_{min} = \gamma_{min} m$, y la energía cinética es $T = (\gamma_{min} - 1) m$:
   para el electrón, $T \approx 0.52 \times 0.511 = 0.27$ MeV; para el muón,
   $T \approx 0.52 \times 105.7 = 55$ MeV.
3. Con $\beta \to 1$, $\cos\theta = 1/1.33$ y $\theta = 41^\circ$.
4. No. Un neutrino de $7$ MeV no tiene energía ni para **crear** el muón ($m_\mu = 106$
   MeV), y mucho menos para darle los $55$ MeV del umbral. Por eso los detectores de agua
   ven los neutrinos solares solo por dispersión con electrones, y por eso el umbral
   experimental de SK está en unos pocos MeV: por debajo, ni el electrón emite luz.

</details>

#### Fotosensores: el PMT

Casi todo lo anterior —centelleadores, Cherenkov, calorímetros— acaba produciendo **luz**,
y hay que convertirla en señal eléctrica. El instrumento clásico es el
**fotomultiplicador** (PMT): el fotón arranca un electrón de un fotocátodo por efecto
fotoeléctrico, y ese electrón se multiplica en una cascada de dínodos a potencial creciente
hasta dar una señal medible, con ganancias de $10^6$–$10^7$.

| |
| -- |
| <img src="./imgs/det_PMT_drawing.png" width = 550 align = "center">|
| esquema de funcionamiento de un PMT [WK]|

Hoy conviven con los fotosensores de estado sólido (SiPM), compactos e insensibles al campo
magnético, que los están sustituyendo en muchas aplicaciones.

### Los experimentos

#### Experimentos en $4\pi$

En un colisionador la interacción ocurre en un punto y los productos salen en todas
direcciones, así que el detector rodea el punto de colisión: por eso se les llama
experimentos **$4\pi$**. Son cebollas de capas cilíndricas, y ya sabemos en qué orden van:

1. **detector de vértices y de trazas**, lo más cerca posible del haz y lo más ligero
   posible, dentro de un solenoide cuyo ${\bf B}$ va en el eje del cilindro;
2. **calorímetro electromagnético**, que para $e$ y $\gamma$;
3. **calorímetro hadrónico**, que para el resto de hadrones;
4. **cámaras de muones**, fuera de todo, a menudo intercaladas en el hierro que cierra el
   campo magnético.

ATLAS, CMS, ALICE en el LHC; ALEPH, DELPHI, L3 y OPAL en LEP; Belle-II en SuperKEKB.
Imágenes de [CMS](https://home.cern/resources/image/experiments/cms-images-gallery).

#### Experimentos de blanco fijo o hacia delante

En un experimento de blanco fijo el centro de masas se mueve, y todos los productos salen
**hacia delante**, en un cono estrecho. No tiene sentido rodear nada: el detector se monta
como una secuencia de estaciones a lo largo del haz, con un imán **dipolar** que desvía las
trazas en un plano.

El orden de los subdetectores es el mismo; lo que cambia es la geometría. Es también la
configuración de los experimentos *hacia delante* en un colisionador, como LHCb, diseñado
para física del quark $b$, que se produce con ángulos pequeños.

|  |
|:-- |
| <img src="./imgs/det_LHCb_subdetectors.jpg" width = 500 align = "center"> |
| esquema de [LHCb](https://lhcb.cern/detector) en el LHC|

### Del dato al resultado

**¿Qué ocurre entre el sensor y la figura de un artículo?**

En el LHC hay un cruce de paquetes cada $25$ ns y el detector tiene $\mathcal{O}(10^8)$
canales: son petabytes por segundo, varios órdenes de magnitud por encima de lo que se
puede grabar. La primera decisión del experimento es, por tanto, **qué tirar**.

* El **sistema de disparo** (*trigger*) decide en tiempo real qué sucesos se guardan. En
  ATLAS y CMS trabaja en dos niveles: uno *hardware*, con electrónica dedicada y una
  lectura parcial del detector, que baja de $40$ MHz a $\sim 100$ kHz en pocos
  microsegundos; y otro *software*, una granja de miles de CPU que reconstruye el suceso
  parcialmente y baja a $\mathcal{O}(1)$ kHz.

* El **sistema de adquisición** (DAQ) lee, sincroniza y ensambla los fragmentos de los
  distintos subdetectores en un único suceso, y lo escribe.

Trigger y DAQ forman el ***online***. Lo que el trigger descarta se pierde para siempre:
es la decisión más delicada del diseño de un experimento.

Después, y ya sin prisa, viene el ***offline***: la **reconstrucción**. De las señales de
los sensores se suben peldaños sucesivos:

$$
\text{señales} \to \text{puntos} \to \text{trazas y depósitos} \to \text{vértices}
\to \text{partículas} \to \text{cuadrimomentos}
$$

Son programas de C++ y Python en los que participan decenas o centenares de físicos, y que
se ejecutan en centros de computación distribuidos por todo el mundo (el *Grid*).

En paralelo se simulan **millones de sucesos** por Monte-Carlo: se genera la física según
el modelo, se propagan las partículas por una descripción detallada del detector, se
simula la respuesta de cada sensor y se reconstruye **con el mismo programa** que los datos
reales. Sin esa simulación no hay eficiencias, no hay aceptancias y no hay medida.

#### El análisis

Con los sucesos reconstruidos, el análisis selecciona los que corresponden al proceso que
se busca y estima el observable. Aquí lo que manda es la **estadística**: técnicas de
clasificación (hoy, redes neuronales), ajustes de máxima verosimilitud, y un cuidado
extremo con lo que se usa de control —datos de calibración, regiones de control en los
propios datos, simulación—.

Hay dos escuelas estadísticas, la **frecuentista** (la habitual en el campo) y la
**bayesiana**, y la diferencia importa a la hora de interpretar un intervalo.

El resultado se presenta de una de estas dos formas.

* Una **estimación** del observable —una sección eficaz, una vida media, una masa, una
  fracción de desintegración— con dos errores:

  - **estadístico**, que depende del número de sucesos y baja como $1/\sqrt{N}$;
  - **sistemático**, que recoge lo que no se conoce con exactitud: calibración,
    eficiencias, modelos teóricos, luminosidad. No baja tomando más datos, y acaba siendo
    el que limita las medidas maduras.

  Por ejemplo, [la observación del Higgs en ATLAS](https://arxiv.org/abs/1207.7214).

* Un **límite**, cuando se busca algo y no se encuentra: la vida media del protón, la masa
  del neutrino, la sección eficaz de un proceso nuevo. Se da con un **nivel de confianza**
  (el $90\,\%$ es lo habitual) y **sin errores**, porque las incertidumbres ya están
  incorporadas en el propio límite.

  Por ejemplo, el límite en la masa del neutrino de
  [KATRIN](https://arxiv.org/abs/2406.13516).

Lo que significa exactamente ese intervalo depende del método estadístico empleado: es la
misma discusión que vimos al poner el límite a la desintegración del protón en
Super-Kamiokande.

### Recapitulación

De la partícula al número, en cinco pasos:

| | |
|:--|:--|
| el acelerador **produce** | $\sqrt{s}$ y $\mathcal{L}$, con la pared de $E^4/m^4$ |
| la partícula **interacciona** | ioniza ($\mathrm{d}E/\mathrm{d}x$), radia ($X_0$) o choca ($\lambda_I$) |
| el detector **recoge** | trazas y $p_T = 0.3B\rho$; calorímetros y $\sigma_E/E \propto 1/\sqrt{E}$; Cherenkov y $\beta$ |
| las capas **identifican** | dónde para cada partícula dice qué era |
| el análisis **mide** | trigger, reconstrucción, simulación, y un número con su error o un límite |

Este proceso nos permite estimar un observable, pero ¿cómo lo predecimos? Lo veremos en la sección de la perspectiva teórica.

## Una perspectiva teórica

**¿De dónde sale el número que el experimento mide?**

La perspectiva experimental terminaba en una sección eficaz o una vida media **medidas**. La
teórica empieza justo ahí: el mismo número, **calculado**. El camino tiene cinco peldaños:

1. los **ingredientes**: teoría cuántica de campos y cinemática;
2. el **núcleo** del cálculo: la regla de oro y el elemento de matriz;
3. la ecuación de **Dirac**, que nos da el fermión y su corriente;
4. la **teoría de Fermi**: cuatro fermiones en un punto;
5. los **diagramas de Feynman** y el propagador.

Es una visión, no una deducción. Las demostraciones y los detalles están en
[ext-dirac](./ext-dirac.ipynb) y en [ext-fundamentos](./ext-fundamentos.ipynb).

### Los ingredientes: teoría cuántica de campos y cinemática

**¿Qué hace falta para predecir una sección eficaz?**

Dos cosas, y conviene separarlas desde el principio:

| | |
|:--|:--|
| la **cinemática** | *cuántos* estados finales caben: el **espacio fásico** |
| la **dinámica** | *qué* interacciona y con qué intensidad: el **elemento de matriz** $M_{fi}$ |

La cinemática ya la tenemos del tema anterior: masas, $\sqrt{s}$, momentos en el centro de
masas. La dinámica es lo nuevo, y su marco es la **teoría cuántica de campos** (TCQ).

#### Por qué campos y no funciones de onda

La mecánica cuántica asocia una función de ondas $\psi(x)$ a *una* partícula. Eso no basta
aquí, por dos razones:

* en una colisión las partículas **se crean y se destruyen**: el número de partículas no es
  constante;
* la Naturaleza fabrica partículas **idénticas** —el protón de un átomo de hidrógeno y el de
  un rayo cósmico de otra galaxia son el mismo objeto—.

En TCQ el objeto fundamental es el **campo**, uno por cada partícula fundamental, y las
partículas son sus **excitaciones**: un operador crea una partícula sobre el vacío, y otro
crea su antipartícula, con las cargas opuestas.

$$
|p, s, q\rangle = a^\dagger_{p,s} \, |0\rangle, \qquad
|p, s, \bar{q}\rangle = b^\dagger_{p,s} \, |0\rangle
$$

Es la idea de los operadores escalera del oscilador armónico, con la partícula como cuanto
de excitación del campo. Que los operadores fermiónicos **anticonmuten** es, ni más ni menos,
el principio de exclusión de Pauli.

Cada espín tiene su ecuación de campo libre:

| campo | ecuación | |
|:--|:--|:--|
| escalar, $S = 0$ | $(\partial^\mu\partial_\mu + m^2) \, \phi(x) = 0$ | Klein-Gordon |
| espinorial, $S = 1/2$ | $(i\gamma^\mu \partial_\mu - m) \, \Psi(x) = 0$ | Dirac |
| vectorial, $S = 1$ | $\partial_\mu ( \partial^\mu A^\nu - \partial^\nu A^\mu ) = 0$ | Maxwell |

Libres, esos campos no hacen nada. La **interacción** aparece al exigir que la teoría sea
invariante bajo un cambio de fase **local**, $\Psi(x) \to e^{i q \theta(x)} \Psi(x)$: para
que lo sea hay que sustituir la derivada por la **derivada covariante**,

$$
\partial_\mu \; \to \; D_\mu \equiv \partial_\mu + i q A_\mu(x)
\qquad \Rightarrow \qquad
(i \gamma^\mu D_\mu - m) \, \Psi(x) = 0
$$

y eso **obliga** al fermión a acoplarse con el campo $A_\mu$. La invariancia *gauge* fija la
interacción y lleva asociada una cantidad conservada: en electromagnetismo, la carga
eléctrica. Volveremos sobre ello en el tema del Modelo Estándar.

### Sección eficaz y vida media: el elemento de matriz

**¿Dónde está exactamente la física?**

El puente entre la teoría y lo que se mide es la **regla de oro de Fermi**: la frecuencia de
transición de un estado inicial $| i \rangle$ a uno final $\langle f |$ es

$$
\nu = 2\pi \, |M_{fi}|^2 \, \rho(E), \qquad M_{fi} = \langle f | H_{int} | i \rangle
$$

y tiene dos factores de naturaleza muy distinta:

* $\rho(E)$, la **densidad de estados**, que es **solo cinemática**;
* $M_{fi}$, el **elemento de matriz**, que es **toda la dinámica**.

Todo lo que viene después de esta transparencia —Dirac, Fermi, Feynman— es maquinaria para
calcular $M_{fi}$.

Resuelta la parte cinemática, los dos casos que nos interesan quedan así, en el centro de
masas (donde $^*$ marca las cantidades):

| | |
| :--: | :--: |
| $\Gamma(a \to b + c)$ | $\sigma(a + b \to c + d)$ |
| <img src="./imgs/intro_drawing_decay2.png" width = 230 align="center"> | <img src="./imgs/intro_drawing_int2.png" width = 230 align="center"> |

$$
\Gamma = \frac{p^*}{32 \pi^2 m^2_a} \int_\Omega |M_{fi}|^2 \, \mathrm{d}\Omega^*,
\qquad
\sigma = \frac{1}{64 \pi^2 s} \, \frac{p^*_f}{p^*_i} \int_\Omega |M_{fi}|^2 \,
\mathrm{d}\Omega^*
$$

**¿Qué ves?** Las dos tienen la misma estructura: un factor cinemático conocido —masas,
momentos en el CM, $s = (E_a + E_b)^2$— multiplicando la integral de $|M_{fi}|^2$. La vida
media y la sección eficaz del tema anterior son, del lado teórico, la misma cuenta con
distinto envoltorio.

[Ver ext-fundamentos](./ext-fundamentos.ipynb), [MT3.3, MT3.4]

### La ecuación de Dirac

**¿Qué es un fermión, para poder escribir su corriente?**

Dirac buscó en 1928 una ecuación relativista **lineal** en las derivadas —la temporal y las
espaciales en pie de igualdad— cuyo cuadrado devolviera $E^2 - {\bf p}^2 = m^2$:

$$
\boxed{ (i \gamma^\mu \partial_\mu - m) \, \Psi(x) = 0 }
$$

El precio de esa linealidad es que los coeficientes $\gamma^\mu$ no pueden ser números: son
cuatro matrices complejas $4\times4$ que cumplen el álgebra
$\{\gamma^\mu, \gamma^\nu\} = 2 g^{\mu\nu}$, y $\Psi(x)$ tiene **cuatro componentes**.

El pago es lo que la convierte en una de las cumbres de la física teórica: de ella salen,
sin haberlos puesto a mano, el **espín $1/2$** y las **antipartículas**.

[Ver ext-dirac](./ext-dirac.ipynb) para las matrices $\gamma$, la deducción de los espinores
y la quiralidad.

#### Las soluciones: los espinores

Las soluciones son ondas planas, cuatro: dos de **fermión** y dos de **antifermión**, con dos
estados de espín cada una,

$$
\Psi_s(x) = u_s(p) \, e^{-i(Et - {\bf p}{\bf x})}, \qquad
\Phi_s(x) = v_s(p) \, e^{+i(Et - {\bf p}{\bf x})}, \qquad s = 1, 2
$$

Toda la dependencia en $(t, {\bf x})$ está en la fase; los **espinores** $u_s(p), v_s(p)$
—cuatro componentes en columna— dependen solo de $p = (E, {\bf p})$.

| |
| :--: |
| <img src="./imgs/dirac_Escher_Moebius.jpg" width = 240 align="center"> |
| hormiga en una cinta de Möbius, M. C. Escher |

El espinor es un objeto nuevo: ni escalar ni vector. Se llama así porque hay que girarlo
$4\pi$ —dos vueltas— para recuperarlo, como la hormiga de Escher, que recorre la cinta dos
veces para volver al punto de partida.

#### Espín

De la ecuación de Dirac se sigue que el momento angular orbital **no** se conserva,
$[H, {\bf L}] \neq 0$, pero sí lo hace ${\bf J} = {\bf L} + {\bf S}$, con

$$
{\bf S} \equiv \frac{1}{2} {\bf \Sigma}, \qquad
\Sigma_k = \begin{pmatrix} \sigma_k & 0 \\ 0 & \sigma_k \end{pmatrix}
$$

Es decir: el fermión tiene un momento angular **intrínseco** de valor $1/2$, y no ha habido
que postularlo. Los dos espinores $u_1, u_2$ son sus proyecciones en $z$:

$$
S_3 \, u_1(p) = + \frac{1}{2} \, u_1(p), \qquad S_3 \, u_2(p) = - \frac{1}{2} \, u_2(p)
$$

#### Antipartículas

Al aplicar los operadores de energía y momento a las soluciones $\Phi_s(x)$ salen $-E$ y
$-{\bf p}$: **energías negativas**. La lectura física es que $\Phi_s$ describe la
**antipartícula**, con la misma masa y el mismo espín que la partícula, y **todas las cargas
opuestas**.

| |
| :--: |
| <img src="./imgs/obs_pos_discovery.png" width = 260 align="center"> |
| descubrimiento del positrón, Anderson 1932 [WP] |

Anderson lo vio en una cámara de niebla dentro de un campo magnético: una traza cuya
curvatura y pérdida de energía eran las de un "electrón positivo". Cuatro años después de la
ecuación.

Las dos interpretaciones clásicas de la energía negativa se quedan cortas:

* el **mar de Dirac**: en el vacío todos los estados de energía negativa están ocupados, y el
  positrón es un *hueco*. Funciona —y reaparece en los pares electrón-hueco de los
  semiconductores—, pero exige un vacío infinitamente cargado y un espectro no acotado por
  abajo;

* **Feynman y Stückelberg**: la antipartícula es una partícula que viaja hacia atrás en el
  tiempo, porque $t \to -t$ convierte una fase en la otra. Buena regla de cálculo, física
  incómoda.

La respuesta correcta es la de TCQ: $b^\dagger$ **crea** una antipartícula de energía
positiva, y los operadores dan directamente los valores físicos. Otra razón para mirar la
ecuación de Dirac como una ecuación de campo y no de función de ondas.

#### Las soluciones, en tres casos

Con ${\bf p}$ en la dirección $z$, los cuatro espinores son

$$
u_{1} = N \begin{pmatrix} 1 \\ 0 \\ \kappa \\ 0 \end{pmatrix}, \;\;
u_{2} = N \begin{pmatrix} 0 \\ 1 \\ 0 \\ -\kappa \end{pmatrix}, \;\;
v_{1} = N \begin{pmatrix} 0 \\ -\kappa \\ 0 \\ 1 \end{pmatrix}, \;\;
v_{2} = N \begin{pmatrix} \kappa \\ 0 \\ 1 \\ 0 \end{pmatrix}, \qquad
\kappa = \frac{\mathrm{p}}{E+m}, \;\; N = \sqrt{E+m}
$$

Todo está en $\kappa$, que mide **cuán relativista** es la partícula:

| caso | $\kappa$ | qué queda |
|:--|:--:|:--|
| en reposo, $\mathrm{p} = 0$, $m > 0$ | $0$ | solo las dos componentes de arriba: el espinor de Pauli de siempre |
| en movimiento, $m > 0$ | $0 < \kappa < 1$ | el *boost* "enciende" las dos componentes de abajo |
| sin masa, o $E \gg m$ | $\to 1$ | arriba y abajo pesan igual |

**¿Qué ves?** Las dos componentes de abajo son el precio de la relatividad: aparecen al poner
el fermión en movimiento y dominan en el límite ultrarrelativista, que es justo donde trabaja
la física de partículas.

#### Helicidad

Sin un eje externo —un campo magnético— el único eje disponible es el del propio momento. La
proyección del espín sobre él es la **helicidad**:

$$
h \equiv \frac{{\bf S} \cdot {\bf p}}{\mathrm{p}} = \frac{1}{2}
\begin{pmatrix} \hat{\bf v} \cdot \vec{\sigma} & 0 \\ 0 & \hat{\bf v} \cdot \vec{\sigma} \end{pmatrix}
$$

Conmuta con el hamiltoniano, luego se conserva, y en la base de helicidad los cálculos a alta
energía se simplifican mucho. Para ${\bf p}$ en $z$, los espinores de antes **son** los de
helicidad, $u_\pm(p), v_\pm(p)$.

| |
|:--:|
|<img src="./imgs/dirac_spinors_helicities.png" width = 400 align="center">|
| espín (azul) y momento (negro) de los espinores de helicidad |

Con una salvedad decisiva: si la partícula tiene masa, la helicidad **no es invariante
Lorentz** —siempre hay un sistema que adelanta a la partícula e invierte su momento—. Solo
para $m = 0$ es un buen número cuántico en cualquier sistema. Esto será central en la
interacción débil.

#### La corriente fermiónica

De todo lo anterior, lo que de verdad se usa en un cálculo es una sola cosa. Con el espinor
adjunto $\bar{\Psi} \equiv \Psi^\dagger \gamma^0$ se construyen bilineales con propiedades
Lorentz definidas:

| | | |
| :--: | :--: | :--: |
| norma | $\bar{\Psi}(x) \Psi(x)$ | escalar |
| **corriente** | $\bar{\Psi}(x) \gamma^\mu \Psi(x)$ | **cuadrivector** |

La corriente $j^\mu = \bar{\Psi}\gamma^\mu\Psi$ cumple $\partial_\mu j^\mu = 0$: es una
corriente de probabilidad conservada y, multiplicada por la carga, la corriente eléctrica.

Ese cuadrivector es la pieza con la que se construyen **todos** los elementos de matriz de
aquí en adelante. [Ver ext-dirac](./ext-dirac.ipynb), [MT4.2]

### Interacciones a cuatro fermiones: la teoría de Fermi

**¿Cómo se escribe una interacción con lo que ya tenemos?**

Fermi (1934) tenía delante la desintegración $\beta$,

$$
n \to p + e + \bar{\nu}_e
$$

cuatro fermiones en un mismo punto y ningún mediador a la vista. Su propuesta:

| |
| :--: |
| <img src="./imgs/intro_fermi_currents.png" width = 270 align="center"> |
| el acoplo puntual entre dos corrientes |

* la interacción es **puntual**, y su intensidad la fija una constante, $G_F$;
* intervienen los espinores de las **cuatro** partículas;
* $M_{fi}$ debe ser **invariante Lorentz**, y para eso lo natural es el producto escalar de
  dos corrientes.

Con la corriente hadrónica y la leptónica,

$$
j^\mu_{had} = \bar{\Psi}_p \gamma^\mu \Psi_n, \qquad
j^\nu_{lep} = \bar{\Psi}_e \gamma^\nu \Psi_{\bar{\nu}_e}
$$

el elemento de matriz es su producto escalar:

$$
M_{fi} \propto \frac{G_F}{\sqrt{2}} \; j_{had} \cdot j_{lep} =
\frac{G_F}{\sqrt{2}} \left[ \bar{\Psi}_p \gamma^\mu \Psi_n \right] g_{\mu\nu}
\left[ \bar{\Psi}_e \gamma^\nu \Psi_{\bar{\nu}_e} \right]
$$

con $G_F = 1.17 \cdot 10^{-5}$ GeV$^{-2}$, la **constante de Fermi**. Dos observaciones:

* le rechazaron el artículo, y aun así la teoría describe bien la desintegración $\beta$;

* $G_F$ tiene **dimensiones** de $1/E^2$, y eso es la firma de una teoría **efectiva**: vale
  a baja energía y se rompe a alta. Lo que falta, justamente, es el **propagador**.

### Diagramas de Feynman y el propagador

**¿Cómo se calcula $M_{fi}$ sin empezar de cero en cada proceso?**

Feynman, en los años 50, juntó tres ideas —la regla de oro, las dos corrientes de Fermi y el
mediador de Yukawa— en una representación gráfica con **reglas de cálculo exactas**, válidas
para todas las fuerzas:

| |
|:--:|
|<img src="./imgs/feynman_default.png" width = 270 align="center">|
| diagrama de una dispersión $a + b \to c + d$ |

| elemento | qué es | qué aporta a $M_{fi}$ |
|:--|:--|:--|
| **corriente** | la línea del fermión, de $a$ a $c$ | $\bar{\Psi}_c \gamma^\mu \Psi_a$ |
| **vértice** | el acoplo | un factor $g$ (dos por diagrama: $M_{fi} \propto g^2$) |
| **mediador** | la línea interna, $X$ | el **propagador** |

El tiempo corre hacia la derecha y los antifermiones llevan la flecha al revés. Del mediador
no sabemos nada: no se observa, es **virtual**.

#### El propagador

El propagador es el factor que aporta el mediador, y depende **solo** del cuadrimomento
transferido $q$:

| | electromagnética | débil cargada | débil neutra | fuerte |
|:--|:--:|:--:|:--:|:--:|
| acoplo | $e$ | $g_W$ | $g_Z$ | $g_s$ |
| propagador | $\frac{g_{\mu\nu}}{q^2}$ | $\frac{g_{\mu\nu}}{q^2 - m^2_W}$ | $\frac{g_{\mu\nu}}{q^2 - m^2_Z}$ | $\frac{g_{\mu\nu}}{q^2}$ |

con $m_W = 80.4$ GeV y $m_Z = 91.2$ GeV. El tensor $g_{\mu\nu}$ es lo que cose las dos
corrientes, igual que en la teoría de Fermi.

**¿Qué ves?** La masa del mediador **en el denominador**. Eso es lo que hace que una fuerza
tenga alcance corto y parezca débil a baja energía; un mediador sin masa ($\gamma$, gluón) da
$1/q^2$ y alcance infinito.

Para comparar intensidades conviene usar constantes **adimensionales**, $\alpha = g^2 / 4\pi$;
en electromagnetismo, la constante de estructura fina $\alpha = e^2/4\pi \simeq 1/137$:

| electromagnética | débil | fuerte |
| :--: | :--: | :--: |
| $\alpha \simeq 1/137$ | $\alpha_W \simeq 1/30$ | $\alpha_S \sim 1$ |

Con dos vértices por diagrama, $M_{fi} \propto g^2 \propto \alpha$, y la probabilidad
$|M_{fi}|^2 \propto \alpha^2$. **En esta escala la interacción débil no es débil**: su
debilidad a baja energía viene del propagador, no del acoplo.

En cada vértice se conservan la **carga eléctrica** (el $W^\pm$ se la lleva), el
**cuadrimomento** ($q = p_a - p_c$) y el **sabor** —salvo en el $W^\pm$, el único que cambia
$f \to f'$, y siempre entre los dos elementos de un doblete—.

#### Un ejemplo completo

| |
| :--: |
| <img src="./imgs/feynman_emuemu_momenta_dot.png" width = 300 align="center"> |
| dispersión $e + \mu \to e + \mu$ mediada por un fotón |

Juntando las reglas —dos corrientes, dos vértices y un propagador—:

$$
M_{fi} \propto e^2 \, \frac{j_e \cdot j_\mu}{q^2} =
e^2 \left[ \bar{\Psi}_e(p_c) \gamma^\mu \Psi_e(p_a) \right] \frac{g_{\mu\nu}}{q^2}
\left[ \bar{\Psi}_\mu(p_d) \gamma^\nu \Psi_\mu(p_b) \right], \qquad q = p_a - p_c
$$

Compáralo con el de Fermi: es la misma estructura —corriente $\cdot$ corriente— con el
propagador en medio en lugar de una constante.

#### Tres tipos de diagrama

| |
| :--: |
|<img src="./imgs/feynman_new_type_interactions.png" width = 460 align="center">|
| dispersión, aniquilación y desintegración |

| dispersión | aniquilación | desintegración |
| :--: | :--: | :--: |
| $q = p_a - p_c$ | $q = p_a + p_b$ | $q = p_a - p_c$ |

En la aniquilación el mediador se lleva **toda** la energía, $q^2 = s > 0$, y puede ponerse
*on-shell*: por eso un $e^+ + e$ a $\sqrt{s} = m_Z$ produce $Z$ a espuertas. En la dispersión
$q^2 < 0$ y el mediador es siempre virtual.

*Cuestión*: construye un diagrama de dispersión mediado por un $W$ con un $\nu_e$ en una de
las ramas.

#### La desintegración $\beta$, otra vez

| |
| :--: |
|<img src="./imgs/feynman_new_beta_decay.png" width = 280 align="center">|
| $n \to p + e + \bar{\nu}_e$: un quark $d$ se convierte en $u$ y emite un $W$ |

Los otros dos quarks son espectadores. Como aquí
$q^2 \sim \mathcal{O}(\mathrm{MeV}^2) \ll m^2_W$, el propagador se vuelve una constante:

$$
\frac{g_{\mu\nu}}{q^2 - m^2_W} \; \longrightarrow \; - \frac{g_{\mu\nu}}{m^2_W}
$$

y el diagrama se reduce a un acoplo puntual de dos corrientes: **la teoría de Fermi es el
límite de baja energía de la interacción débil**. Comparando ambas,

$$
\frac{G_F}{\sqrt{2}} = \frac{g^2_W}{8 \, m^2_W}
$$

<div class="admonition tip">
<p class="admonition-title"><b>[>] Taller — ¿Por qué la interacción débil es débil?</b></p>

Datos: $G_F = 1.17 \cdot 10^{-5}$ GeV$^{-2}$, $m_W = 80.4$ GeV, $\alpha = 1/137$.

1. Obtén $g_W$ a partir de $G_F$ y $m_W$, y compáralo con la carga eléctrica,
   $e = \sqrt{4\pi\alpha}$.

2. Estima la razón entre la amplitud débil y la electromagnética,

$$
\frac{|M_W|}{|M_\gamma|} \sim \frac{g^2_W}{|q^2 - m^2_W|} \, \frac{q^2}{e^2}
$$

   en una desintegración nuclear, $q^2 \sim (1 \, \mathrm{MeV})^2$, y en el LHC,
   $q^2 \sim (200 \, \mathrm{GeV})^2$.

3. ¿Quién es entonces el responsable de que la débil sea débil?
</div>

<details>
<summary><b>Solución</b></summary>

1. $g_W = m_W \sqrt{8 G_F / \sqrt{2}} = 80.4 \times \sqrt{6.6 \cdot 10^{-5}} \approx 0.65$,
   frente a $e = \sqrt{4\pi/137} = 0.30$. **El acoplo débil es el doble que el
   electromagnético.**

2. El cociente es $(g^2_W/e^2) \times q^2/|q^2 - m^2_W| \approx 4.6 \times
   q^2/|q^2 - m^2_W|$:

   - a $q^2 = 10^{-6}$ GeV$^2$: $4.6 \times 10^{-6}/6500 \approx 7 \cdot 10^{-10}$;
   - a $q^2 = 4 \cdot 10^{4}$ GeV$^2$: $4.6 \times 4\cdot10^4 / 3.4\cdot10^4 \approx 5$.

3. El **propagador**, no el acoplo. Nueve órdenes de magnitud de supresión en la
   desintegración $\beta$ salen enteros del factor $q^2/m^2_W$; cuando $q^2$ se acerca a
   $m^2_W$ esa supresión desaparece y las dos interacciones son comparables. "Débil" es una
   afirmación sobre la escala de energía, no sobre la fuerza.

</details>

#### Árbol y lazo

Todos los diagramas anteriores son de **nivel árbol**. Al mismo proceso contribuyen también
diagramas con lazos:

| |
| :--: |
| <img src="./imgs/feynman_tree_loop.png" width = 400 align="center"> |
| árbol (izda.) y lazo (dcha.) en $e + e \to e + e$ |

El de árbol va como $M_{fi} \propto \alpha$, y el de lazo como $\alpha^2$. Si el acoplo es
pequeño, el cálculo se ordena como una **serie perturbativa** y el árbol es el término
dominante (*leading order*). **En este curso nos quedaremos siempre en nivel árbol.**

Para $\alpha_S \sim 1$ ese argumento deja de valer, y ahí empieza la dificultad de la
interacción fuerte.

### Recapitulación teórica

De la teoría al número, en cinco pasos:

| | |
|:--|:--|
| el marco **cuantiza campos** | las partículas son excitaciones; las antipartículas, sus compañeras de carga opuesta |
| la regla de oro **separa** | cinemática, $\rho(E)$, de dinámica, $M_{fi}$ |
| Dirac **da el fermión** | espín $1/2$, antipartículas, helicidad y la corriente $\bar{\Psi}\gamma^\mu\Psi$ |
| Fermi **acopla dos corrientes** | $G_F$, puntual y con dimensiones: una teoría efectiva |
| Feynman **añade el mediador** | vértices ($g$) y propagador ($1/(q^2 - m^2)$), y reglas para todas las fuerzas |

Con esto se calculan $\sigma$ y $\Gamma$ de cualquier proceso a nivel árbol, que es lo que
haremos en los temas siguientes.

## Bibliografía

 * [AB] A. Bettini, *Introduction to Elementary Particle Physics*, Cambridge U. Press.
   Temas 1.10, 1.11, 1.12 (aceleradores y detectores), 1.1, 1.3, 1.5 y 2.5, 2.6
   (diagramas de Feynman).

 * [MT] M. Thomson, *Modern Particle Physics*, Cambridge U. Press. Temas 1.2, 1.3 y
   apéndice D; y, para la parte teórica, 3.3 y 3.4 (regla de oro), 4 (ecuación de
   Dirac) y 5, 6 (reglas de Feynman y QED).

 * [MS] B. R. Martin, G. Shaw, *Particle Physics*, Wiley. Temas 2.1, 2.2.

 * [PDG](https://pdg.lbl.gov/) Particle Data Group: *Passage of particles through matter*
   y *Particle detectors at accelerators*, las dos referencias de trabajo del campo.

 * C. Grupen, B. Shwartz, *Particle Detectors*, Cambridge U. Press, para quien quiera
   entrar en serio.

 * [Peskin] M. Peskin, *Lectures on the Theory of the Weak Interaction*, 2016
   CERN-JINR European School of Particle Physics,
   [arXiv:1708.09043](https://arxiv.org/abs/1708.09043).


## Créditos de las imágenes

**Con fuente indicada en el pie**: **[PDG]**, Particle Data Group; **[AB]**, el libro de
Bettini; **[ALICE]**, **[DELPHI]**, **[CMS]**, **[LHCb]** y **[SK]**, esas colaboraciones;
**[CERN]** y **[Fermilab]**, esos laboratorios; **[WK]**, Wikimedia Commons.

Las figuras de los esquemas de detectores enlazan a la página del experimento
correspondiente, donde están sus condiciones de uso.

El esquema del momento transverso perdido es de elaboración propia.

De la parte teórica: el descubrimiento del positrón (Anderson, 1932) **[WP]**;
la cinta de Möbius, M. C. Escher. Los diagramas
de Feynman y los esquemas de corrientes son de elaboración propia.
